<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaSelectDialog</class>
 <widget class="QDialog" name="MediaSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Multimedia</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="imageButton">
     <property name="text">
      <string>Image</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../../../../../../usr/share/yappari/icons/48x48/80-59.png</normaloff>../../../../../../../../../../../usr/share/yappari/icons/48x48/80-59.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="audioButton">
     <property name="text">
      <string>Audio</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../../../../../../usr/share/yappari/icons/48x48/85-1.png</normaloff>../../../../../../../../../../../usr/share/yappari/icons/48x48/85-1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="videoButton">
     <property name="text">
      <string>Video</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../../../../../../usr/share/yappari/icons/48x48/80-61.png</normaloff>../../../../../../../../../../../usr/share/yappari/icons/48x48/80-61.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
